kernel/src/process_loading.rs

```diff
@@ -273,7 +273,6 @@ fn load_processes_from_flash<C: Chip, D: ProcessStandardDebug + 'static>(
                         }
 
                         // Skip this binary and move to the next one.
-                        continue;
```

Comment:
This one is less bad than the kernel loop, largely by its location, but it's again a case where I think there is value in the robustness of the codebase to preserve the explicit control flow directive here, rather than implicit behavior of the current structure and this 'happening-to -be-at-the-end'.